src/hotspot/share/opto/superword.cpp

```diff
@@ -1886,9 +1831,8 @@ bool SuperWord::profitable(Node_List* p) {
     Node* second_in = p0->in(2);
     Node_List* second_pk = my_pack(second_in);
     if ((second_pk == nullptr) || (_num_work_vecs == _num_reductions)) {
-      // Unmark reduction if no parent pack or if not enough work
+      // No parent pack or not enough work
       // to cover reduction expansion overhead
-      _loop_reductions.remove(p0->_idx);
```

Comment:
Note: "unmarking" reductions is not necessary, I think it has no effect either way. We will never pack this reduction any more anyway.
I want to remove this, so that the reductions logic in `VLoopReduction` can be read-only in SuperWord.